src/pages/workspaces/List.js

```diff
@@ -226,15 +228,17 @@ export const WorkspaceList = ajaxCaller(class WorkspaceList extends Component {
             div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, [
               'Workspaces'
             ]),
-            div({ style: styles.toolbarButtons }, [
+            div({ style: { ...styles.toolbarButtons, marginLeft: 'auto', display: 'flex' } }, [
```

Comment:
This wrapper and the one below now only contain a single element each, so they're unnecessary. The margins could go directly on the `Clickable`.